src/components/Acknowledgment/Acknowledgment.module.scss

```diff
@@ -0,0 +1,59 @@
+@import "../../styles/variables";
+
+.acknowledgment_container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-left: 5px;
+
+  .label {
+    text-transform: uppercase;
+    font-weight: 700;
+    color: $blue;
+    margin-right: 5px;
+  }
+
+  .names_container {
+    display: flex;
```

Comment:
Let's fix the alignment of the name and the button. `align-items: center;` didn't work for me. But I was adjusting it in the inspector. What did work was just setting the link to `position: relative;`, then a -3px margin.

Reply:
This did not work. (I fidgeted with this for almost an hour during the first draft of the code).
However, I targeted the SVG instead. Not the prettiest solution, but c'est la vie.